server/src/live-pricing.js

```diff
@@ -39,13 +34,13 @@ const createSession = async (params) => {
     });
     if (response.status !== STATUS_CODES.CREATED) {
       const json = await response.json();
-      throw new Error(json);
+      throw json;
```

Comment:
I didn't realise you could throw anything and it would create an error with that value, nice! How is it different from what was there before though?

Reply:
So before it was wrapping the json in an `Error`, so when it's surfaced to the user and it's printed, they just see `[Object object]`, which wasn't so helpful. Here's a code snippet to illustrate:

![screen shot 2018-09-04 at 11 57 12](https://user-images.githubusercontent.com/73652/45027592-bc164280-b039-11e8-943c-e423d69bc33a.png)

Reply:
Looks good! Thanks!

Reply:
I don't fully get what you mean, do you mean do it like this:

```
throw new Error(JSON.stringify(json))
```

?

Reply:
That would work ... ideally we would extract the failure _reason_ from the response i.e. `json.message` but I appreciate that our api response doesn't make it easy to obtain that and `JSON.stringify(json)` is probably "good enough".

Reply:
Yeah I think this will be more robust for any errors that come up. As it's a one liner and we've already agreed on it, I'll yolo it to `master`